tests/run-make/crate-import-export-static/rmake.rs

```diff
@@ -0,0 +1,12 @@
+// Test that a static can be exported from one crate and imported into another.
+//
+// This was broken for Arm64EC as only functions, not variables, should be
+// decorated with `#`.
+// See https://github.com/rust-lang/rust/issues/138541
+
+use run_make_support::rustc;
+
+fn main() {
+    rustc().input("export.rs").panic("abort").run();
```

Comment:
I think you probably want to specify a target here as otherwise, this test will only be executed for Tier 1 target platforms so `arm64ec-pc-windows-msvc` will never be tested against.